src/fragments/start/getting-started/angular/data-model.mdx

```diff
@@ -98,120 +98,135 @@ When prompted, select **GraphQL**. This will open the AWS AppSync console for yo
 
 Open `src/main.ts` and add the following code to configure the Angular project with Amplify:
 
-```javascript
+```typescript
 import Amplify from "aws-amplify";
 import aws_exports from "./aws-exports";
 Amplify.configure(aws_exports);
 ```
 
-Update `tsconfig.app.json` to include the "node" compiler option in *types*:
+<Callout>
+  On Angular 12+, Typescript{" "}
+  <a href="https://www.typescriptlang.org/tsconfig#strict" target="_blank">
+    strict mode
+  </a>{" "}
+  is enabled by default, which has a known issue with importing `aws-exports.js`.
+  You can resolve this by adding an <code>aws-exports.d.ts</code> to <code>src/</code> (
+  <a
+    href="https://gist.github.com/wlee221/1b02f519e6e0ce25d3643cf2948f4467"
```

Comment:
NON-BLOCKING: In the future, I believe we need to make a gist on a org account in case we need to update via the team.

Reply:
Agreed, there are other gists on docs that should be shared in org.